chia/server/node_discovery.py

```diff
@@ -166,12 +169,29 @@ async def on_connect(peer: ws.WSChiaConnection):
 
         await self.server.start_client(self.introducer_info, on_connect)
 
+    async def _query_dns(self, dns_address):
+        try:
+            peers: List[TimestampedPeerInfo] = []
+            result = dns.resolver.query(dns_address, "A")
```

Comment:
isn't this async?
DNS lookups can hang for 2 minutes in the worst case. I think it's important to not make a blocking call.

Also, shouldn't we look up AAAA records as well?
Even though the server doesn't support it yet, I expect we'll add IPv6 support at some point